travis/script_deploy_android_snapshot.sh

```diff
@@ -3,7 +3,16 @@
 set -e
 set -o pipefail
 
-if [ "${PLATFORM}" = "android" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then
+if [ "${PLATFORM}" = "android" ] && [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "mult-android-arch" ]; then
+    
```

Comment:
@tallytalwar can we rename the branch to android-deploy? Since we will have to push to this branch to deploy the android libs.